Tests/Indicators/AugenPriceSpikeTests.cs

```diff
@@ -42,6 +42,13 @@ public void TestWithStream()
             }
         }
 
+        [Test]
+        public void TestForPeriod()
+        {
+            Assert.Throws<ArgumentException>(() => new AugenPriceSpike(1));
+            Assert.Throws<ArgumentException>(() => new AugenPriceSpike(2));
```

Comment:
These test might be better as a `[TestCase(1)] & [TestCase(2)]` of the same unit test